ide/coqide/coqOps.ml

```diff
@@ -859,10 +859,11 @@ object(self)
   method get_n_errors =
     Doc.fold_all document 0 (fun n _ _ s -> if has_flag s `ERROR then n+1 else n)
 
-  method get_errors =
+  method get_errors_warnings =
     let extract_error s =
-      match List.find (function `ERROR _ -> true | _ -> false) s.flags with
-      | `ERROR (loc, msg) ->
+      match List.find (function `ERROR _ | `WARNING _ -> true | _ -> false) s.flags with
```

Comment:
Is it possible for sentences to get both a warning and an error? If so this might set the wrong tag, or it might lose information. I think you should use List.find_all instead.

Reply:
Yes, I think so.  Updated the code.

Reply:
We could even have multiple warnings and an error.